src/views/Results.vue

```diff
@@ -240,9 +269,9 @@ export default class Results extends ResultsProps {
   /* This is supposed to cause the dialog to automatically appear if there is
    * no file uploaded
    */
-  mounted() {
+  /*mounted() {
```

Comment:
When removing code if it is no longer needed it should just be deleted, not commented out. If it is ever needed in the future again the git history can be used to find it.